staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -93,6 +93,13 @@ func NewEventFromRequest(req *http.Request, requestReceivedTimestamp time.Time,
 	return ev, nil
 }
 
+func maybeReplaceVerb(attributes authorizer.Attributes, req *http.Request) string {
+	if len(attributes.GetVerb()) == 0 {
+		return fmt.Sprintf("unrecognized(%v)", req.Method)
+	}
+	return attributes.GetVerb()
```

Comment:
naive and curiosity question, is it better to assign `attributes.GetVerb()`  to a variable and execute the method only once, or execute `GetVerb()` twice, or just "don't mind the go compiler will solve it"

Reply:
aaaah good point. I'm not sure the Go compiler is clever enough to optimize this on the fly. Let me create a temporary variable then.